src/Text/Pandoc/Readers/Org/Meta.hs

```diff
@@ -190,6 +192,15 @@ parseFormat = try $ replacePlain <|> replaceUrl <|> justAppend
    rest            = manyTill anyChar         (eof <|> () <$ oneOf "\n\r")
    tillSpecifier c = manyTill (noneOf "\n\r") (try $ string ('%':c:""))
 
+excludedTagSet :: Monad m => OrgParser m (Set.Set Tag)
+excludedTagSet = do
+  skipSpaces
+  ((Set.fromList . map Tag) <$>
```

Comment:
I believe this will fail if there is trailing whitespace after the last tag. A common idiom is to use `many1 (foo *> skipSpaces1) *> newline`.

Reply:
I believe some of the parentheses here could be omitted.

Reply:
Thanks. I used `many [...] skipSpaces [...]` instead since a blank input to `#+EXCLUDE_TAGS` in Emacs results in `noexport` no longer being an excluded tag.